README.md

```diff
@@ -13,23 +13,19 @@
   - [Register Workflow](#register-workflow)
   - [Run Workflow](#run-workflow)
   - [Run Task Worker](#run-task-worker)
-- [Advanced Topics](#advanced-topics)
-  - [Inspect the TaskRun](#inspect-the-taskrun)
-  - [Search for Someone's Workflow](#search-for-someones-workflow)
-  - [NodeRuns and TaskRuns](#noderuns-and-taskruns)
-  - [Debugging Errors](#debugging-errors)
+  - [Posting an Event](#posting-an-event)
 - [Next Steps](#next-steps)
 
-**Get started in under 5 minutes, or your money back!** :wink:
+**Get started in under 5 minutes, or your money back!** 😉
 
-This repo contains a minimal example to get you started using LittleHorse in Java. [LittleHorse](www.littlehorse.dev) is a high-performance orchestration engine which lets you build workflow-driven microservice applications with ease.
+This repo contains a minimal example to get you started using [LittleHorse](https://littlehorse.io) in Java. [LittleHorse](https://littlehorse.io) is a high-performance orchestration engine which lets you build workflow-driven microservice applications with ease.
 
 You can run this example in two ways:
 
 1. Using a local deployment of a LittleHorse Server (instructions below, requires one `docker` command).
 2. Using a LittleHorse Server deployed in a cloud sandbox (to get one, contact `info@littlehorse.io`).
```

Comment:
the wording on 2 is a little weird no reason to say server or deployed.  Suggest:
2.  Using a LittleHorse cloud sandbox.